magento-engcom/import-export-improvements
# SKU uniqueness validation

### Description
Removed logic for generating unique SKU when already exists. Throw an exception instead which will be presented to the user.

### Fixed Issues (if relevant)
https://github.com/magento-engcom/import-export-improvements/issues/101

### Manual testing scenarios
1. Create product with sku "TEST"
2. Create another product with sku "TEST"
3. You should receive an error saying that the sku already exists instead of getting your sku modified.

### Contribution checklist
